test/labeling/model/test_label_model.py

```diff
@@ -209,6 +209,13 @@ def test_predict_proba(self):
         np.testing.assert_array_almost_equal(probs, true_probs)
 
     def test_predict(self):
+        L = np.array([[-1, 1, 0], [0, -1, 1], [1, 0, -1]])
```

Comment:
Can we add a simple comment here that this test is confirming that with 3 LFs that counteract one another results in tie votes and therefore abstains on all points?